app/controllers/v1/greetings_controller.rb

```diff
@@ -14,7 +14,7 @@ def index
         greeting: 'Hello, Peter'
       },
       {
-        greeting: 'Buenas, buenaaaaaaaaaaaaaaaaaas'
+        greeting: 'Bonjour'
       }
     ] }.to_json
   end
```

Comment:
- [ ] Kindly, use `db seed` to keep the sample data instead of implementing it inside the controller.